dist.sh

```diff
@@ -70,16 +78,27 @@ for platform in $platforms; do
 
     cp -r $template_dir/* $dist_path/
 
-    cp $core_dir/build/libs/*.jar $dist_path/workcraft.jar
+    # resource_dir is simply the distribution root. OS X needs special care.
+    resource_dir=$dist_path
+    if [ "$platform" = "osx" ]; then
+        contents_dir=$resource_dir/Contents
+        resource_dir=$contents_dir/Resources
 
-    mkdir -p $dist_path/plugins
+        # Update Info.plist with version
+        echo "$(sed "s/__VERSION__/$version/" $contents_dir/Info.plist)" \
```

Comment:
`sed` has an `-i` option that runs the command in the file, greatly simplifying your command.

```sh
sed -i "s/__VERSION__/$version/" $contents_dir/Info.plist
```